help/de/Help_ch_GUIMenus.xml

```diff
@@ -490,7 +502,13 @@
 
               <row>
                 <entry>
-                  <para><guimenuitem>Drucken...</guimenuitem>
+                  <para><menuchoice>
+                      <shortcut>
+                      <keycombo>
+                        <keycap>Strg</keycap><keycap>P</keycap>
+                      </keycombo>
+                      </shortcut><guimenuitem>Drucken...</guimenuitem>
```

Comment:
Kein Accellerator?

Reply:
_Drucken (Ctrl+P) ist wohl schon in gnome-application definiert, bei uns aber inaktiv und wird von Spezialisierungen "{Bericht|Rechnung|…} drucken" überschrieben.
also `<accell>D</accell>rucken`